products/dashboard/.env.example

```
@@ -0,0 +1,13 @@
+NEXT_PUBLIC_SENTRY_DSN=https://<public-key>@o<org>.ingest.sentry.io/<project-id>
+SENTRY_DSN=https://<public-key>@o<org>.ingest.sentry.io/<project-id>
+
+# Recommended runtime metadata for meaningful Sentry filtering and regressions
+NEXT_PUBLIC_SENTRY_ENVIRONMENT=development
+SENTRY_ENVIRONMENT=development
+NEXT_PUBLIC_SENTRY_RELEASE=local-dev
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SENTRY_RELEASE key should go before the SENTRY_ENVIRONMENT key

(UnorderedKey)